dolphinscheduler-meter/src/main/resources/grafana/DolphinSchedulerWorker.json

```diff
@@ -172,7 +172,7 @@
         "showThresholdLabels": false,
         "showThresholdMarkers": false
       },
-      "pluginVersion": "8.5.3",
+      "pluginVersion": "9.0.0-beta2",
```

Comment:
I don't recommend using a beta version of plugin

Reply:
Sure, will fix it.

Reply:
Fixed by the latest commit.